python/ray/serve/api.py

```diff
@@ -456,6 +457,7 @@ def decorator(_func_or_class):
     return decorator(_func_or_class) if callable(_func_or_class) else decorator
 
 
+@deprecated(instructions="Check out https://docs.ray.io/en/latest/serve/index.html")
```

Comment:
```suggestion
@deprecated(instructions="Please see https://docs.ray.io/en/latest/serve/index.html")
```

Reply:
same goes for others. existing one is very informal tone, reads a bit funny to me